cmd/kube-apiserver/app/server.go

```diff
@@ -221,8 +222,29 @@ func CreateKubeAPIServerConfig(opts options.CompletedOptions) (
 		generatedopenapi.GetOpenAPIDefinitions,
 	)
 	if err != nil {
-		return nil, nil, nil, err
+		return nil, nil, nil, fmt.Errorf("failed to create generic config: %w", err)
+	}
+
+	// generic controlplane admission initializers
+	controlPlaneAdmissionConfig := &controlplaneadmission.Config{
+		ExternalInformers:    versionedInformers,
+		LoopbackClientConfig: genericConfig.LoopbackClientConfig,
+	}
+	serviceResolver := buildServiceResolver(opts.EnableAggregatorRouting, genericConfig.LoopbackClientConfig.Host, versionedInformers)
+	pluginInitializers, err := controlPlaneAdmissionConfig.New(proxyTransport, genericConfig.EgressSelector, serviceResolver, genericConfig.TracerProvider)
```

Comment:
just nit on naming maybe? `controlPlaneInitializers` and `kubeInitializers` ? Or was there any deeper idea behind the current name `pluginInitializers?

Reply:
This code moves in the follow-up. Would not touch it again now, if not needed.